<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageDetails</class>
 <widget class="QWidget" name="MessageDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MessageDetails</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Message</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="nameLabel">
          <property name="text">
           <string>name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="nameEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="idLabel">
          <property name="text">
           <string>id</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="idEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="dlcLabel">
          <property name="text">
           <string>dlc</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="dlcEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="frequencyLabel">
          <property name="text">
           <string>frequency</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="frequencyEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="descriptionLabel">
          <property name="text">
           <string>description</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="descriptionEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="signalContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>298</width>
           <height>60</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2"/>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="addButton">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteMessageButton">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
